### DISCUSSION TITLE
Filter for teaser selection

### DISCUSSION THREAD

**Discussion by Vavavenya** [sulu/sulu#5794] [Q&A]
How I can limit variant for select for each page with teaser_selection?
I tried set provider and type(like for snippet_selection) like params
For example on this page I want pick only stories and hide another variants
sulu version 2.0
![image](https://user-images.githubusercontent.com/39053900/106902896-606ca980-670a-11eb-98a9-8989f62ebd91.png)

**Comment by niklasnatter** [comment]
Hey,
thanks for reaching out! Unfortunately, it is not possible to limit the providers for a specific `snippet_selection` property at the moment. In your case, it might be an alternative to just a `page_selection` if you want the content manager to select pages only.

Feel free to create a feature request or contribute a pull request in this repository. If this is a critical feature for you, it is also possible to sponsor the development. Just ping us via the [contact form on our website](https://sulu.io/contact-us) if you are interested in that.
